src/inmanta/protocol/auth.py

```diff
@@ -213,12 +215,16 @@ def decode_token(token: str) -> claim_type:
                             )
                     decoded_payload[k] = v
                 case _:
-                    logging.getLogger(__name__).info(
-                        "Only claims of type string or list of strings are supported. %s is filtered out.", k
-                    )
+                    unsupported.append(k)
+
+        if unsupported:
+            logging.getLogger(__name__).debug(
+                "Only claims of type string or list of strings are supported. %s are filtered out.", ", ".join(unsupported)
+            )
 
         ct_key = const.INMANTA_URN + "ct"
-        decoded_payload[ct_key] = [x.strip() for x in str(payload[ct_key]).split(",")]
+        ct_value = str(payload.get(ct_key, "api"))
+        decoded_payload[ct_key] = [x.strip() for x in ct_value.split(",")]
```

Comment:
Add a default client type (api) so we can support JWT providers that do not allow custom claims (like cloudflare does)